roles/edpm_ovn/templates/kolla_ovn_controller.yaml.j2

```diff
@@ -1,8 +1 @@
-command: "/usr/bin/ovn-controller --pidfile --log-file unix:/run/openvswitch/db.sock {% if edpm_ovn_tls_enabled | bool %} -p /etc/pki/tls/private/ovndb.key -c /etc/pki/tls/certs/ovndb.crt -C /etc/pki/tls/certs/ovndbca.crt {% endif %}"
-permissions:
-  - path: /var/log/openvswitch
-    owner: root:root
-    recurse: true
-  - path: /var/log/ovn
-    owner: root:root
-    recurse: true
+command: "/usr/bin/ovn-controller --pidfile unix:/run/openvswitch/db.sock {% if edpm_ovn_tls_enabled | bool %} -p /etc/pki/tls/private/ovndb.key -c /etc/pki/tls/certs/ovndb.crt -C /etc/pki/tls/certs/ovndbca.crt {% endif %}"
```

Comment:
is it ok not providing the pidfile value?

Reply:
I don't know, I don't think the pid file created by ovn-controller is even used anywhere. We use the systemd service pidfile:
 PIDFile=/run/ovn_controller.pid

Reply:
From the man page:
       --pidfile[=pidfile]
              Causes a file (by default, program.pid) to be created
              indicating the PID of the running process. If the pidfile
              argument is not specified, or if it does not begin with /,
              then it is created in .

So it seems like it's ok to not provide a path